static/app/utils/useMetricsContext.tsx

```diff
@@ -0,0 +1,16 @@
+import {useContext} from 'react';
+
+import {MetricsContext} from 'sentry/utils/metrics/metricsContext';
+
+export function useMetricsContext() {
+  const metricsContext = useContext(MetricsContext);
+
+  if (!metricsContext) {
+    throw new Error('useMetricsContext was called outside of MetricsContextProvider');
+  }
+
+  return {
+    metricTags: metricsContext.tags,
+    metricMetas: metricsContext.metas,
+  };
```

Comment:
I would avoid remapping here and just return
```suggestion
  return metricsContext
```

Whoever is going to use this can always
`const {tags: metricTags, metas: metricMetas} = useMetricsContext()`

Reply:
yes that is the only disadvantage

Reply:
just updated it :)